src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -247,7 +247,9 @@ class SeekBar extends Slider {
     super.handleMouseUp(event);
 
     // Stop event propagation to prevent double fire in progress-control.js
-    event.stopPropagation();
+    if (event) {
```

Comment:
why do we end up calling this function without an `event`?

Reply:
https://github.com/videojs/video.js/blob/270a23196a524045db726e099ba1d713026236c2/src/js/control-bar/progress-control/progress-control.js#L130

Reply:
And then: https://github.com/videojs/video.js/blob/270a23196a524045db726e099ba1d713026236c2/src/js/control-bar/progress-control/progress-control.js#L191

Reply:
ah, I was trying to find where this could happen but only found cases that pass an `event` object through, but I guess there's a possibility of it not existing.